### ISSUE TITLE
WebGL Queries return error

### ISSUE THREAD

**Issue by alexhaloETH** [dojoengine/dojo.unity#34]
**What is happening and Code**
When making a query in a WebGL build the code returns the error 
```
Uncaught Error: unknown variant `model`, expected one of `Keys`, `Member`, `Composite` at line 1 column 39
``` 
Whereas if done in the editor there is no error that is returned.

The function used in the query is a simple modification of the synchronization function thats already in the unity sdk just with the addition of a keys clause
```
        public async Task<int> CustomSynchronizeEntities(string modelName, string[] keys)
        {
            var query = new dojo.Query
            {
                clause = new dojo.COptionClause
                {
                    tag = dojo.COptionClause_Tag.SomeClause,
                    some = new dojo.Clause
                    {
                        tag = dojo.Clause_Tag.Keys,
                        keys = new dojo.KeysClause
                        {
                            model = modelName,
                            keys = keys
                        }
                    },
                },
                limit = 1,
                offset = 0,
            };

#if UNITY_WEBGL && !UNITY_EDITOR
            var entities = await worldManager.wasmClient.Entities(query);
#else
            var entities = await Task.Run(() => worldManager.toriiClient.Entities(query));
#endif

            var entityGameObjects = new List<GameObject>();
            foreach (var entity in entities)
            {
                entityGameObjects.Add(SpawnEntity(entity.HashedKeys, entity.Models.Values.ToArray()));
            }

            OnSynchronized?.Invoke(entityGameObjects);
            return entities.Count;
        }
``` 

**Expected Behaviour**
The WebGl build should query fine and synchronize the entity fine without erroring out

**Comment by Xaxxoo** [comment]
Hi @alexhaloETH can I try this with a quick solution please?

**Comment by Larkooo** [comment]
This is still WIP as queries are not yet fully supported and interoperable on WebGL which uses the wasm bindings. Eventually we'd want a nice way to share interfaces and types between native and webgl so queries become much more simpler to make for both targets.

**Comment by tuong-student** [comment]
```
        public ToriiClient(string toriiUrl, string rpcUrl, string relayUrl, string world)
        {
            CString ctoriiUrl = CString.FromString(toriiUrl);
            CString crpcUrl = CString.FromString(rpcUrl);
            CString crelayUrl = CString.FromString(relayUrl);
            CString cworld = CString.FromString(world);

            Debug.Log("torii: " + toriiUrl);
            Debug.Log("rpc: " + rpcUrl);
            Debug.Log("relay: " + relayUrl);
            Debug.Log("world: " + world);
            var result = dojo.client_new(ctoriiUrl, crpcUrl, crelayUrl, cworld, (dojo.KeysClause*)0, (UIntPtr)0);
            Debug.Log("result: " + result.tag);
            
            if (result.tag == dojo.ResultToriiClient_Tag.ErrToriiClient)
            {
                throw new Exception(result.err.message);
            }

            client = result._ok;

            RegisterEntityStateUpdateEvent(new dojo.FieldElement[] { });
            RegisterEventMessageUpdateEvent(new dojo.FieldElement[] { });
        }
```

My Debug.Log("result: " + result.tag) always return error, even though I have the correct URLs and world address. My torii is running too.

**Comment by alexhaloETH** [comment]
@tuong-student Hey dude, i think the issue you are having is different from the main issue opened here, the issue opened is a known error that hopefully will be fixed with the 1.0 release. As for your problem, i think it would be more helpfull if you join the Dojo server and speak in the unity channel, maybe i can give you a hand
